### DISCUSSION TITLE
Looking to add partner to post dogs

### DISCUSSION THREAD

**Discussion by miyares** [sheltermanager/asm3#1431] [Ideas]
We wanted to see if we could automatically post our partner shelters' dogs to [PetPair](www.petpair.co/shelters). The organization has been getting our dogs to adoptive homes, but they currently require a new form for each dog and we'd love for it to automate posting from ShelterManager like it does with Petfinder. How do we get that added?

**Comment by Tetzor** [comment]
Hi,

We have now replied to your email with the information on this.

Kind regards,

Jon
